app/controllers/search_controller.rb

```diff
@@ -18,6 +18,17 @@ def show
     end
   end
 
+  # JSON API for Searchworks' mini-bento
```

Comment:
This is doing the exact same thing as the show function. Is there a reason show can't be used instead of having to literally repeat the exact same code?

Reply:
It's not doing the same. This has a hardcoded lib_guides service and the other takes a service passed by parameter.

Reply:
Well the parameter is already being passed by SearchWorks and the original implementation of this used the endpoint parameter so is there a reason we have to basically the same thing twice?

Reply:
The search version also returns html and this returns JSON.  It uses a different route scheme. I don't think the benefit of reducing three duplicate lines of code is worth the cost to readability.